release/community-stable/parrot/docker/Dockerfile

```diff
@@ -23,10 +23,10 @@ ARG PS_INSTALL_VERSION=6
 ADD ${PS_PACKAGE_URL} /tmp/powershell.deb
 
 # Define Args for the needed to add the package
-ARG DEBIAN_PACKAGE_URL=http://ftp.us.debian.org/debian/pool/main/i/icu/libicu57_57.1-6+deb9u2_amd64.deb
+ARG DEBIAN_PACKAGE_URL=http://ftp.us.debian.org/debian/pool/main/i/icu/libicu57_57.1-6+deb9u3_amd64.deb
```

Comment:
Maybe in the future we should put these kind of files in some kind of long term filestorage so they can't just go missing when deleted from upstream - Azure?

Reply:
Two options I see, 
Create a docker or nuget package registry.
https://github.com/features/package-registry

for docker, create an image, and copy the file from the image.

for nuget, download, unzip and get the package.

I've already asked the team for DEB and RPM support.

Reply:
I agreed with @TravisEz13 on the advantage of the current system.

I think the "caching packages/libraries or not" discussion should probably be done in the Issue section and not on this PR. Do you agreed? @RDIL 

Until we have consensus on whether we cache the package approach or not, let get this PR move forward for now.